com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/message/Messages.java

```diff
@@ -439,4 +439,6 @@
     public static final String SERVICES_IN_PROGRESS = "Services in progress: {0}";
     public static final String BUILD_FOR_PACKAGE_0_ALREADY_EXISTS = "Build for package: \"{0}\" already exists";
 
+    protected Messages() {

```

Comment:
private?

Reply:
The constructor is protected because this class is overrided